### ISSUE TITLE
Zooming function for wide format video recording doesn't work correctly

### ISSUE THREAD

**Issue by LeopardSurd** [CyanogenDefy/android_device_motorola_jordan#59]
Hallo!
Like the title says, there are problems whith zooming for wide format video recording. When I want to zoom from the menu, the zooming effect is incorrect. It distorts the image but don't zoom. 

Tested on 11 July nightly. 

Green lens.

**Comment by AL2** [comment]
In the testing I've made with it (same nightly), I did not get any distorted images, but the zoom selection slider sometimes doesn't match the current zoom. 

The zoom slider can show being at level 6 but no zoom shown on screen --> that happens if you zoom in, go check a previously taken picture and come back to camera: slider is still up but zoom is reset at 0. 
In all cases though, just changing zoom level to a different value replaces things.

But my phone has red lens (Bayer) camera though - so sorry, I didn't want to spam your issue report....just wanted to also report that little zoom glitch I'm having without creating a new issue.
